python/ray/tests/test_actor.py

```diff
@@ -273,7 +272,7 @@ def __init__(self):
             pass
 
     Foo.remote()
-
+    # TODO: redis-removal kv
```

Comment:
Just confirming my understanding, this is the definition of the actor that the function manager puts in redis right?

Reply:
Yes, and it's not handled in this PR.